app/parse.py

```diff
@@ -14,5 +19,30 @@ class Course:
     type: CourseType
 
 
+def parse_single_course(product_soup: BeautifulSoup, type_: str) -> Course:
+    return Course(
+        name=product_soup.select_one("span.typography_landingH3__vTjok").text,
+        short_description=product_soup.select_one("p").text,
+        type=CourseType(type_)
```

Comment:
read about enum
`type_` here is `CourseType.FULL_TIME` or `CourseType.PART_TIME`
`CourseType(type_)` is incorrect